posthog-react-native/package.json

```diff
@@ -34,7 +35,8 @@
     "expo-device": ">= 4.0.0",
     "expo-file-system": ">= 13.0.0",
     "expo-localization": ">= 11.0.0",
-    "react-native-device-info": ">= 10.0.0"
+    "react-native-device-info": ">= 10.0.0",
+    "react-native-navigation": ">=6.0.0"
```

Comment:
This should be added as optional to `peerDependenciesMeta` as well, so that users without `react-native-navigation` don't have to add it.

Reply:
Ah yep